typings/internal-electron.d.ts

```diff
@@ -13,6 +13,7 @@ declare namespace Electron {
   }
 
   interface App {
+    _isSecureModeEnabled(): boolean;
```

Comment:
I think this should be exposed as real API to allow third party Electron module to know if secure mode is enabled

Reply:
should I make it a readonly property?

Reply:
is it possible to make a property `readonly` in `electron.d.ts`?